base/MAPL_LocStreamMod.F90

```diff
@@ -329,6 +338,30 @@ subroutine MAPL_LocStreamGet(LocStream, NT_LOCAL, TILETYPE, TILEKIND, &
        tilegrid = locstream%Ptr%TILEGRID
     end if
 
+    if (present(local_i)) then
+#ifdef __GFORTRAN__
```

Comment:
Hmm. I wonder if we just shouldn't add the `#ifdef __GFORTRAN__` bits here? We are planning to remove them in #716

Reply:
@mathomp4  I agree, the #ifdef __GFORTRAN__ is not needed anymore. @bena-nasa could you simply remove this little section

Reply:
Removed all gfortran references.